app/controllers/posts_controller.rb

```diff
@@ -5,4 +5,21 @@ def index
     posts = Post.all
     render status: :ok, json: { posts: }
   end
+
+  def create
+    post = Post.new(post_params)
+    post.save!
```

Comment:
```suggestion
    Post.create!(post_params)
```

This is much cleaner.